crates/ruma-common/tests/api/ruma_api.rs

```diff
@@ -1,13 +1,10 @@
 #[test]
 fn ui() {
     let t = trybuild::TestCases::new();
-    t.pass("tests/api/ui/01-api-sanity-check.rs");
-    t.compile_fail("tests/api/ui/02-invalid-path.rs");
```

Comment:
Where did the invalid path test go?

Reply:
Ah I wanted to leave a note about this. `trybuild` doesn't seem usable for const evaluation error reporting, as the compiler includes the full path to my toolchain directory when pointing to the `panic` macro in std. Since endpoint history now goes through `VersionHistory::new`, we can use regular runtime tests (not sure whether we have any currently).

Reply:
Ah, please do make an issue about that, so that we can track and re-add the test when that (would be) resolved on trybuild's side.